docs/core/introduction.md

```diff
@@ -179,231 +76,172 @@ A .NET app is built from source code by using [MSBuild](/visualstudio/msbuild/ms
 
-For more information, see [Managed execution process](../standard/managed-execution-process.md) and [Tiered compilation](whats-new/dotnet-core-3-0.md#tiered-compilation).
+The .NET Team at Microsoft works collaboratively with other organizations to distribute and support .NET in various ways.
 
-### AOT compiler
+[Red Hat supports .NET](https://developers.redhat.com/topics/dotnet/) on Red Hat Enterprise Linux (RHEL).
 
-The default experience for most .NET workloads is the JIT compiler, but .NET offers two forms of ahead-of-time (AOT) compilation:
+[Samsung supports .NET](https://developer.tizen.org/development/training/.net-application) on Tizen platforms.
 
-* Some scenarios require 100% AOT compilation. An example is [iOS](/xamarin/ios/).
-* In other scenarios, most of an app's code is AOT-compiled but some is JIT-compiled. Some code patterns aren't friendly to AOT (like generics). An example of this form of AOT compilation is the [ready-to-run](whats-new/dotnet-core-3-0.md#readytorun-images) publish option. This form of AOT offers the benefits of AOT without its drawbacks.
+## Runtime technology
 
-### Automatic memory management
+The [Common Language Runtime (CLR)](../standard/clr.md) provides capabilities and services to apps that govern their function and also define their basic security and reliability behavior. The [fundamental features of the runtime](https://github.com/dotnet/runtime/blob/main/docs/design/coreclr/botr/intro-to-clr.md) are:
 
-The *garbage collector* (GC) manages the allocation and release of memory for applications. Each time your code creates a new object, the CLR allocates memory for the object from the [managed heap](../standard/garbage-collection/fundamentals.md#the-managed-heap). As long as address space is available in the managed heap, the runtime continues to allocate space for new objects. When not enough free address space remains, the GC checks for objects in the managed heap that are no longer being used by the application. It then reclaims that memory.
+* Garbage collection
+* Memory safety and type safety
+* High level support for programming languages
+* Cross-platform design
 
-The GC is one of the CLR services that help ensure *memory safety*. A program is memory safe if it accesses only allocated memory. For instance, the runtime ensures that an app doesn't access unallocated memory beyond the bounds of an array.
+.NET is sometimes called a "managed code" runtime. It is called *managed* primarily because it uses a garbage collector for memory management and because it enforces type and memory safety. The CLR is also called a *virtual machine*. It virtualizes (or abstracts) a variety of operating system and hardware concepts, such as memory, threads, and exceptions.
```

Comment:
I don't think the cognitive load of 'virtual machine' is worth introducing here. It has a very clear and different meaning in most people's minds. Back when .NET was created, we called it the VM but virtual machines in today's sense were (IIRC) not something most developers were exposed to. Even on our team, some old timers call it the VM but most do not call it that and I have never heard anyone on the team refer to it as a virtual machine.

Reply:
The JVM is the counter example. That said, I can remove this.